### ISSUE TITLE
Reviewing missing roundabouts

### ISSUE THREAD

**Issue by sbelemey** [mapbox/mapping#420]
### Intro
Our team noticed that in some countries there are problems with the mapping of roundabouts and the presence of tags in them. Missed roundabouts - part of the road where there's a roundabout sign, but the roundabout road with needed tags is absent. 
Countries where we'll work: **[Spain](https://wiki.openstreetmap.org/wiki/Organised_Editing/Activities/missing_roundabouts_Spain)**, **[Italy](https://wiki.openstreetmap.org/wiki/Organised_Editing/Activities/missing_roundabouts_Italy)**, **[France](https://wiki.openstreetmap.org/wiki/Organised_Editing/Activities/missing_roundabouts_France)**, **[United Kingdom](https://wiki.openstreetmap.org/wiki/Organised_Editing/Activities/missing_roundabouts_UK)**, **[Belgium](https://wiki.openstreetmap.org/wiki/Organised_Editing/Activities/missing_roundabouts_Belgium)**, [**Brazil**](https://wiki.openstreetmap.org/wiki/Organised_Editing/Activities/missing_roundabouts_Brazil#Examples_of_missing_roundabouts_in_OSM), **[USA](https://wiki.openstreetmap.org/wiki/Organised_Editing/Activities/missing_roundabouts_USA#Examples_of_missing_roundabouts_in_OSM)**, **[Germany](https://wiki.openstreetmap.org/wiki/Organised_Editing/Activities/missing_roundabouts_Germany#Intro)**.

Which items will be reviewed? 
- Missed geometry of roundabout 
- Roundabout without tag 'junction=roundabout/circular'

Hashtags we will use in this project: 
**_#mapbox_roundabouts_UK, #mapbox_roundabouts_Italy, #mapbox_roundabouts_France, #mapbox_roundabouts_Belgium, #mapbox_roundabouts_Spain, #mapbox_roundabouts_Brazil, #mapbox_roundabouts_Germany, #mapbox_roundabouts_USA._**

### Background
As part of on-going work to improve the quality of OpenStreetMap data, we use internal Mapbox data validation jobs that search OSM data for errors for detecting issues related to road network data in Europe. In this project, we are going to review issues in 1 category. Our plan is to review all detected roundabout issues and fix it where possible in OpenStreetMap. 

We use internal Mapbox data validation jobs that search OSM data for errors. The process is completely switched to automatic uploading and searching for data errors. Automation of the process was achieved with internal algorithm which detects mistakes on roundabout mapping and them tags. Our plan is to review a roundabout of the detected issues manually, and fix any valid issues with them.

### Examples of missing roundabouts in OSM
 Let's look at a couple of examples of what we will fix in this project:
**Junction=circular**
 
![image](https://user-images.githubusercontent.com/57706756/208319923-35ec8bf9-302f-48c2-83a5-a8a1e279a952.png)

We see that there is no junction=circular tag here, also circular exits are connected at one point. Here we put the tag junction=circular and fixed mapping of circular exits. 

**Junction=roundabout**

![image](https://user-images.githubusercontent.com/57706756/208320155-e3b568aa-c703-46f0-abf4-52b615ed6d3b.png)

there is no proper **junction=roundabout** tag here, but of the roundabout exits is correct. Here we only put the tag junction=roundabout. 

Also we put the tag **Mini-roundabout**: 
<img width="839" alt="Screenshot 2023-02-15 at 11 20 05 AM" src="https://user-images.githubusercontent.com/57490915/218974321-fd746ae0-9ebe-4280-9231-3c80dd80198d.png">

### Timeframe

We started review of detected issues in the February 2023, and editing OpenStreetMap data to fix valid issues.

### Tools

We will use existing OpenStreetMap editing tools
Editor: iD Satellite Imagery: Bing, Maxar, Esri
Street Level Imagery: Mapillary, Other open sources if available

### Contact Person

We welcome feedback, suggestions, and insights from local mappers. If you have questions about this work, or a specific edit by our team, please reach out to Natalia (natallia.zhouner@mapbox.com). Also you can leave a comment in a changesets.

**Comment by SomeoneElseOSM** [comment]
> Countries where we'll work: Spain, Italy, France, United Kingdom and Belgium.

Presumably you'll follow https://wiki.openstreetmap.org/wiki/Organised_Editing_Guidelines and contact those communities in appropriate places to let them know what's happening?  Presumably there will also be a new entry at https://wiki.openstreetmap.org/wiki/Organised_Editing/Activities ?

**Comment by sonyahilchik** [comment]
@SomeoneElseOSM of course, we contacted all communities and added all of our activities into https://wiki.openstreetmap.org/wiki/Organised_Editing/Activities

**Comment by sbelemey** [comment]
@bfontaine, hello!
We have completed the review stage and did not consider the possibility of maintaining the task through MapRoulette.
